internal/clearingway/handlers.go

```diff
@@ -167,83 +180,83 @@ var ProgCommand = &discordgo.ApplicationCommand{
 }
 
 var ReclearCommand = &discordgo.ApplicationCommand{
-	Name:	"reclears",
+	Name:        "reclears",
 	Description: "Assign or remove reclear roles",
 	Options: []*discordgo.ApplicationCommandOption{
 		{
-			Type: discordgo.ApplicationCommandOptionString,
-			Name: "ultimate",
+			Type:        discordgo.ApplicationCommandOptionString,
+			Name:        "ultimate",
 			Description: "The ultimate you want to reclear",
-			Required: true,
+			Required:    true,
 			Choices: []*discordgo.ApplicationCommandOptionChoice{
 				{
-					Name: "UCoB",
+					Name:  "UCoB",
 					Value: "The Unending Coil of Bahamut (Ultimate)",
 				},
 				{
-					Name: "UWU",
+					Name:  "UWU",
 					Value: "The Weapon's Refrain (Ultimate)",
 				},
 				{
-					Name: "TEA",
+					Name:  "TEA",
 					Value: "The Epic of Alexander (Ultimate)",
 				},
 				{
-					Name: "DSR",
+					Name:  "DSR",
 					Value: "Dragonsong's Reprise (Ultimate)",
 				},
 				{
-					Name: "TOP",
+					Name:  "TOP",
 					Value: "The Omega Protocol (Ultimate)",
 				},
 				// TODO: implement when FRU goes live
 				/*
-				{
-					Name: "FRU",
-					Value: "Futures Rewritten (Ultimate)",
-				},
+					{
+						Name: "FRU",
+						Value: "Futures Rewritten (Ultimate)",
+					},
```

Comment:
not like we need to do this now but we should probably revert this format change from gofmt 🙃